CorsixTH/Lua/graphics.lua

```diff
@@ -18,8 +18,12 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
-local TH = require("TH")
+-- The location of a bundled font relative to this file
+local bundled_path = TheApp:getFullPath({"GoNotoKurrent-Bold.ttf"})
+-- The location of a globally installed font
+local installed_path = "/usr/share/fonts/truetype/arphic/uming.ttc"
```

Comment:
Is this global? The path given is likely only *nix global. It seems little odd why you isolated this path though from the other OS paths.

Reply:
It's for *nix packagers to change, I can make that more clear here or in the wiki. It's likely that *nix packagers will put a font at a global path and Windows packagers will put a font at a local path.